nodejs/eks/cluster.ts

```diff
@@ -229,6 +228,21 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         throw new Error("subnetIds, and the use of publicSubnetIds and/or privateSubnetIds are mutually exclusive. Choose a single approach.");
     }
 
+    // Configure the node group options.
+    const nodeGroupOptions: ClusterNodeGroupOptions = args.nodeGroupOptions || {
+        nodeSubnetIds: args.nodeSubnetIds,
+        nodeAssociatePublicIpAddress: args.nodeAssociatePublicIpAddress,
+        instanceType: args.instanceType,
+        nodePublicKey: args.nodePublicKey,
+        nodeRootVolumeSize: args.nodeRootVolumeSize,
+        nodeUserData: args.nodeUserData,
+        minSize: args.minSize,
+        maxSize: args.maxSize,
+        desiredCapacity: args.desiredCapacity,
+        amiId: args.nodeAmiId,
+        version: args.version,
+    };
```

Comment:
Not strictly necessary - but the validation check you had previously that reported if both `args.nodeGroupOptions` and one of these overlapping options were passed would still be nice here.

Reply:
Added back in!